### DISCUSSION TITLE
Maverick timeline

### DISCUSSION THREAD

**Discussion by bradherman** [infinitered/ignite#1997] [General]
Just curious what sort of timeline magnitude is being targeted for Maverick... I know there's no set date, but are we thinking weeks, months, years?

I'm about to rebuild our application from scratch and if it's something like weeks, or even a few months, I might wait... However if we're looking at 3+ months, then I might just start now and worry about upgrading later.  Thanks for all the hard work!  PS is there a way to contribute/tip the developers?

**Comment by jamonholmgren** [comment]
Hey @bradherman! Thanks for the kind words. 🤗 It was posted in our #happy channel in Slack.

<img width="300" alt="happy channel" src="https://user-images.githubusercontent.com/1479215/183774551-d70c3043-dbff-46cd-a118-a0e20e77f280.png">

My anticipation (no promises) is we'll have a reasonable beta out in the next few weeks. We actually have a lot of IR devs working on it right 
now, so it should go pretty quickly. I hope. :-) I'd probably wait, if I were you. And help us test it out when we're ready for that.

As far as tipping the devs, the best tips we can receive at Infinite Red are to keep us in mind if anyone asks you who's a good consultancy for expert React Native help! That's how we get a ton of our day-to-day consulting work, and how we can continue to justify putting time and effort into Ignite and our other open source.

**Reply by bradherman** [reply]
Thanks so much for the quick response... Sounds like I will focus on some housekeeping for now and will definitely refer any friends/colleagues your way in the future.  Thanks again!
